diff --git a/node_modules/react-native/.DS_Store b/node_modules/react-native/.DS_Store
new file mode 100644
index 0000000..c2f4bb9
Binary files /dev/null and b/node_modules/react-native/.DS_Store differ
diff --git a/node_modules/react-native/ReactAndroid/src/main/jni/first-party/fbjni/.npmignore b/node_modules/react-native/ReactAndroid/src/main/jni/first-party/fbjni/.gitignore
similarity index 100%
rename from node_modules/react-native/ReactAndroid/src/main/jni/first-party/fbjni/.npmignore
rename to node_modules/react-native/ReactAndroid/src/main/jni/first-party/fbjni/.gitignore
diff --git a/node_modules/react-native/ReactCommon/hermes/inspector/tools/msggen/.npmignore b/node_modules/react-native/ReactCommon/hermes/inspector/tools/msggen/.gitignore
similarity index 100%
rename from node_modules/react-native/ReactCommon/hermes/inspector/tools/msggen/.npmignore
rename to node_modules/react-native/ReactCommon/hermes/inspector/tools/msggen/.gitignore
diff --git a/node_modules/react-native/scripts/react-native-xcode.sh b/node_modules/react-native/scripts/react-native-xcode.sh
index ad31b9c..f6d173b 100755
--- a/node_modules/react-native/scripts/react-native-xcode.sh
+++ b/node_modules/react-native/scripts/react-native-xcode.sh
@@ -55,137 +55,3 @@ case "$CONFIGURATION" in
     DEV=false
     ;;
 esac
-
-# Path to react-native folder inside node_modules
-REACT_NATIVE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-# The project should be located next to where react-native is installed
-# in node_modules.
-PROJECT_ROOT=${PROJECT_ROOT:-"$REACT_NATIVE_DIR/../.."}
-
-cd "$PROJECT_ROOT" || exit
-
-# Define entry file
-if [[ "$ENTRY_FILE" ]]; then
-  # Use ENTRY_FILE defined by user
-  :
-elif [[ -s "index.ios.js" ]]; then
-  ENTRY_FILE=${1:-index.ios.js}
-else
-  ENTRY_FILE=${1:-index.js}
-fi
-
-if [[ $DEV != true && ! -f "$ENTRY_FILE" ]]; then
-  echo "error: Entry file $ENTRY_FILE does not exist. If you use another file as your entry point, pass ENTRY_FILE=myindex.js" >&2
-  exit 2
-fi
-
-# Find path to Node
-# shellcheck source=/dev/null
-source "$REACT_NATIVE_DIR/scripts/find-node.sh"
-
-# check and assign NODE_BINARY env
-# shellcheck source=/dev/null
-source "$REACT_NATIVE_DIR/scripts/node-binary.sh"
-
-[ -z "$HERMES_CLI_PATH" ] && HERMES_CLI_PATH="$PODS_ROOT/hermes-engine/destroot/bin/hermesc"
-
-if [[ -z "$USE_HERMES" && -f "$HERMES_CLI_PATH" ]]; then
-  echo "Enabling Hermes byte-code compilation. Disable with USE_HERMES=false if needed."
-  USE_HERMES=true
-fi
-
-if [[ $USE_HERMES == true && ! -f "$HERMES_CLI_PATH" ]]; then
-  echo "error: USE_HERMES is set to true but the hermesc binary could not be " \
-       "found at ${HERMES_CLI_PATH}. Perhaps you need to run 'bundle exec pod install' or otherwise " \
-       "point the HERMES_CLI_PATH variable to your custom location." >&2
-  exit 2
-fi
-
-[ -z "$NODE_ARGS" ] && export NODE_ARGS=""
-
-[ -z "$CLI_PATH" ] && export CLI_PATH="$REACT_NATIVE_DIR/cli.js"
-
-[ -z "$BUNDLE_COMMAND" ] && BUNDLE_COMMAND="bundle"
-
-[ -z "$COMPOSE_SOURCEMAP_PATH" ] && COMPOSE_SOURCEMAP_PATH="$REACT_NATIVE_DIR/scripts/compose-source-maps.js"
-
-if [[ -z "$BUNDLE_CONFIG" ]]; then
-  CONFIG_ARG=""
-else
-  CONFIG_ARG="--config $BUNDLE_CONFIG"
-fi
-
-BUNDLE_FILE="$CONFIGURATION_BUILD_DIR/main.jsbundle"
-
-EXTRA_ARGS=
-
-case "$PLATFORM_NAME" in
-  "macosx")
-    BUNDLE_PLATFORM="macos"
-    ;;
-  *)
-    BUNDLE_PLATFORM="ios"
-    ;;
-esac
-
-if [ "${IS_MACCATALYST}" = "YES" ]; then
-  BUNDLE_PLATFORM="ios"
-fi
-
-EMIT_SOURCEMAP=
-if [[ ! -z "$SOURCEMAP_FILE" ]]; then
-  EMIT_SOURCEMAP=true
-fi
-
-PACKAGER_SOURCEMAP_FILE=
-if [[ $EMIT_SOURCEMAP == true ]]; then
-  if [[ $USE_HERMES == true ]]; then
-    PACKAGER_SOURCEMAP_FILE="$CONFIGURATION_BUILD_DIR/$(basename $SOURCEMAP_FILE)"
-  else
-    PACKAGER_SOURCEMAP_FILE="$SOURCEMAP_FILE"
-  fi
-  EXTRA_ARGS="$EXTRA_ARGS --sourcemap-output $PACKAGER_SOURCEMAP_FILE"
-fi
-
-# Hermes doesn't require JS minification.
-if [[ $USE_HERMES == true && $DEV == false ]]; then
-  EXTRA_ARGS="$EXTRA_ARGS --minify false"
-fi
-
-"$NODE_BINARY" $NODE_ARGS "$CLI_PATH" $BUNDLE_COMMAND \
-  $CONFIG_ARG \
-  --entry-file "$ENTRY_FILE" \
-  --platform "$BUNDLE_PLATFORM" \
-  --dev $DEV \
-  --reset-cache \
-  --bundle-output "$BUNDLE_FILE" \
-  --assets-dest "$DEST" \
-  $EXTRA_ARGS \
-  $EXTRA_PACKAGER_ARGS
-
-if [[ $USE_HERMES != true ]]; then
-  cp "$BUNDLE_FILE" "$DEST/"
-  BUNDLE_FILE="$DEST/main.jsbundle"
-else
-  EXTRA_COMPILER_ARGS=
-  if [[ $DEV == true ]]; then
-    EXTRA_COMPILER_ARGS=-Og
-  else
-    EXTRA_COMPILER_ARGS=-O
-  fi
-  if [[ $EMIT_SOURCEMAP == true ]]; then
-    EXTRA_COMPILER_ARGS="$EXTRA_COMPILER_ARGS -output-source-map"
-  fi
-  "$HERMES_CLI_PATH" -emit-binary $EXTRA_COMPILER_ARGS -out "$DEST/main.jsbundle" "$BUNDLE_FILE"
-  if [[ $EMIT_SOURCEMAP == true ]]; then
-    HBC_SOURCEMAP_FILE="$BUNDLE_FILE.map"
-    "$NODE_BINARY" "$COMPOSE_SOURCEMAP_PATH" "$PACKAGER_SOURCEMAP_FILE" "$HBC_SOURCEMAP_FILE" -o "$SOURCEMAP_FILE"
-  fi
-  BUNDLE_FILE="$DEST/main.jsbundle"
-fi
-
-if [[ $DEV != true && ! -f "$BUNDLE_FILE" ]]; then
-  echo "error: File $BUNDLE_FILE does not exist. This must be a bug with" >&2
-  echo "React Native, please report it here: https://github.com/facebook/react-native/issues"
-  exit 2
-fi
diff --git a/node_modules/react-native/third-party-podspecs/boost.podspec b/node_modules/react-native/third-party-podspecs/boost.podspec
index 2f1fcc4..2377c4e 100644
--- a/node_modules/react-native/third-party-podspecs/boost.podspec
+++ b/node_modules/react-native/third-party-podspecs/boost.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |spec|
   spec.homepage = 'http://www.boost.org'
   spec.summary = 'Boost provides free peer-reviewed portable C++ source libraries.'
   spec.authors = 'Rene Rivera'
-  spec.source = { :http => 'https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2',
+  spec.source = { :http => 'https://sourceforge.net/projects/boost/files/boost/1.76.0/boost_1_76_0.tar.bz2',
                   :sha256 => 'f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41' }
 
   # Pinning to the same version as React.podspec.
